doc/analytics.md

```diff
@@ -0,0 +1,172 @@
+[database schema][schema.rb] and querying the data. The intention is that when
+useful common queries are identified they will be made accessible via API
+endpoints or rake tasks.
+
+Here are some example queries to pull out particular insights.
+
+### How many subscribers does a list have at a particular point in time
+
+```ruby
+Subscription.where(
+    "created_at > ? AND (ended_at IS NULL OR ended_at <= ?)",
+    "2018-06-01",
+    "2018-07-01"
+  ).where(subscriber_list_id: 4194).count
+```
+
+### Lists with most new subscriptions in a time frame
+
+```ruby
+pp Subscription.where(
+    "created_at BETWEEN ? AND ?", "2018-06-01", "2018-07-01"
+  ).group(
+    :subscriber_list_id
+  ).having(
+    "count(*) > 10"
+  ).order(
+    count: :desc
+  ).pluck(:subscriber_list_id, Arel.sql("COUNT(*) AS count"))
+```
+
+### Lists with most ended subscriptions in a time frame
+
+```ruby
+pp Subscription.where(
+    "ended_at BETWEEN ? AND ?", "2018-06-01", "2018-07-01"
+  ).group(
+    :subscriber_list_id
+  ).having(
+    "count(*) > 10"
+  ).order(
+    count: :desc
+  ).pluck(:subscriber_list_id, Arel.sql("COUNT(*) AS count"))
+```
+
+### How many emails did subscribers were generated emails for digest runs
+
+```ruby
+pp DigestRun.where(
+    "date >= ? AND date <= ?", "2018-06-04", "2018-06-10"
+  ).where(
+    range: :daily
+  ).order(date: :asc).pluck(:date, :subscriber_count)
+```
+
+### Which hours are content_changes being created at
+
+```ruby
+pp ContentChange.group("EXTRACT(HOUR FROM created_at)").count
+```
+
+### Which days of the week have the most content changes
+
+```ruby
+pp ContentChange.group("to_char(created_at, 'Day')").count
+```
+
+### How many content changes per subscription list in a timeframe
+
+```ruby
+pp MatchedContentChange.joins(:content_change).joins(:subscriber_list).where(
+  "content_changes.created_at BETWEEN ? AND ?",
+  "2018-06-01",
+  "2018-07-01"
+).group(
+  :subscriber_list_id, "subscriber_lists.title"
+).order("COUNT(*) DESC").limit(10).pluck(
+  :subscriber_list_id, "subscriber_lists.title", Arel.sql("COUNT(*)")
+)
+```
+
+## Analytics through Athena
+
+[Athena][athena] is accessible through the AWS control panel which can be
+accessed by following the [instructions][console-instructions] provided in the
+developer docs. To access the production data you will need to use the
+`govuk-infrastructure-production` account, once there you can head to
+[athena](https://eu-west-1.console.aws.amazon.com/athena) and select the
+`email_alert_api_archive` database.
+
+The data stored in Athena is the email id; when the sending process completed;
+whether the email sent; the email subject of the email; the associations to
+content change and digests; and timestamps for when the email was created and
+archived. The data is arranged by partitions of the date when the email
+finished sending - **It is vastly cheaper and faster to query with partitions**.
```

Comment:
I think some expansion of what partitions means in practice would be useful, something like "You should always have a `WHERE` clause that limits the query to at most 7 days unless you know what you're doing". And possibly explain why it's so much cheaper and faster.